docs/pages/roadmap.js

```diff
@@ -0,0 +1,676 @@
+// @flow strict
+import { type Node } from 'react';
+import { Box, Flex, Icon, Text, TapArea, Tooltip, Link } from 'gestalt';
+import Page from '../components/Page.js';
+import MainSection from '../components/MainSection.js';
+import PageHeader from '../components/PageHeader.js';
+
+const tasks = [
```

Comment:
Super nit and not something to worry about, but it could be nice to move the data into a separate file like RoadmapData.json to make future edits easy

Reply:
+1, would also make this file easier to navigate

Reply:
I'm having challenges making this work and not having Typescript bark at me. I may need help to make this happen or we can backlog as an enhancement.

Reply:
Definitely an enhancement, not a blocker for this to ship. Typescript shouldn't be involved (if it is, you'll need to turn that off in your VS Code), but Flow will complain, which is expected. See an example at `docs/components/ColorPalette.js` — we suppress that error.

Reply:
Thanks! Fixed!